tests/serverless_azure/test_application.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import requests
+import subprocess
+import socket
+import time
+
```

Comment:
Could we expand here to validate the new Azure intrinsic agent attributes and transaction metrics?

Reply:
This is the part that is a bit iffy; to test this I have to use the subprocess module which uses a different process, and the validators only work with the main process (it is the same issue that gunicorn tests have)

Reply:
This has been fixed.